1.data-exploration/scripts/nbconverted/1.model_data_visualization.py

```diff
@@ -287,18 +271,22 @@
 
 
 # Pediatric solid vs liquid tumors
+cancer_types = pediatric_model_df['OncotreePrimaryDisease'].tolist()
+
+liquid = []
+non_liquid = []
+
+for type in cancer_types:
+    if liquid_tumors[0] in type or liquid_tumors[1] in type:
```

Comment:
```suggestion
    if cancer_type in liquid_tumors:
```

Does this work? (make actual change in `.ipynb` file).

Also note that I changed the variable name from `type` to `cancer_type`. `type` is not specific enough, and it is good practice to have readable variable names :)

Reply:
Unfortunately that change won't work with the new data because of the more specific cancer typing. I agree that `type` is not specific enough and will incorporate your suggestion of `cancer_type`. I'm also changing `non_liquid` and `liquid` to `ped_non_liquid` and `ped_liquid` to make the cell as a whole more specific.